mockup/patterns/toolbar/pattern.js

```diff
@@ -244,7 +244,7 @@ define([
           }
         });
 
-      $('body').on('click', function(event) {
+      $('body').off('click').on('click', function(event) {
```

Comment:
This would unregister all events handlers bound to the body on "click" event. Modal dialogs closing on a click outside the dialogue could be a case.
To prevent this, you could factor out the inline callback into a normal function which you pass also the ``off`` method as second argument: https://api.jquery.com/off/

Reply:
Sorry - I've got something wrong: You'd have to pass the callback as a third argument.
Instead do this:
```suggestion
      $('body').off('click', 'FRED-PLEASE-FIND-A-NAME').on('click', 'FRED-PLEASE-FIND-A-NAME', function(event) {
```
please replace 'FRED-PLEASE-FIND-A-NAME'  with a suitable name - quickly looking at the code I don't know what it does.

Reply:
@thet thanks! I've updated the .off().on() calls.  To prevent removing all clicks you can add a .namespace value to the event name itself.  I have changed the generic one with 'click.closetoolbarsubmenus'.  It closes opened submenu's when an editor clicks outside the toolbar in the main content.

There are a few other places where we don't use namespaced but maybe should, I'll propose changed to thosein a separate pull request.  First get this one fixed & released.